ci/test/03_test_script.sh

```diff
@@ -167,17 +167,65 @@ if [ "$RUN_CHECK_DEPS" = "true" ]; then
   "${BASE_ROOT_DIR}/contrib/devtools/check-deps.sh" "${BASE_BUILD_DIR}"
 fi
 
+function get_interfaces()
+{
+  set -o pipefail
+  ifconfig | awk -F ':| ' '/^[^[:space:]]/ { if (!match($1, /^lo/)) { print $1 } }'
+  set +o pipefail
+}
+
+function tcpdump_file()
+{
+  echo "/tmp/tcpdump_$1_$2"
+}
+
+function traffic_monitor_begin()
+{
+  test_name="$1"
+  for ifname in $(get_interfaces) ; do
+    tcpdump -nU -i "$ifname" -w "$(tcpdump_file "$test_name" "$ifname")" &
+  done
+}
+
+function traffic_monitor_end()
```

Comment:
nit: here I would also like a bit of documention, maybe adding a small section in `ci/readme.md` would be the right place for this?